db/schema.rb

```diff
@@ -540,6 +540,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer "replies_policy", default: 0, null: false
+    t.boolean "is_exclusive", default: false
```

Comment:
I'm sorry to ask that since Hometown already has this column included, but I believe it should be called `exclusive` without the `is_` prefix.